docs/Credits.md

```diff
@@ -632,6 +632,7 @@ Unless otherwise specified, all the following are from [the Noun Project](https:
 * [Hourglass](https://thenounproject.com/search/?q=hourglass&i=142268) by I Create Stuff for the 'Turn' icon
 * [Shield](https://thenounproject.com/search/?q=shield&i=813568) by Gregor Cresnar for Religious Strength
 * [skill sword flame](https://thenounproject.com/term/skill-sword-flame/2360212/) by Maxicons) for Remove Heresy
+* [Pencil](https://thenounproject.com/search/?q=pencil&i=4195852) by Muhamad Aldi Maulana for Enter Text Prompt Button / Pencil
```

Comment:
Yeah, but somehow - just slightly - I feel it's a bit off the average Unciv style. We're klunkier. And this is... Office. Isn't there some quill in policies that would do? Just a though, feel free to ignore.